docs/researcher/README.md

```diff
@@ -1,3 +1,17 @@
-# Researcher
+---
+home: true
+title: Researcher Documentation
+heroImage: /assets/roar-logo.svg
+actions:
+  - text: Data Tools
+    link: /researcher/data-tools/
+    type: secondary
+  - text: Data Organization
+    link: /researcher/data-organization/
+    type: secondary
+  - text: Data Requests
+    link: /researcher/data-requests/
+    type: secondary
+---
 
-Landing page for researchers
+<p style="text-align: center;">This section provides technical documentation for researchers contributing to ROAR. To suggest edits, please open a pull request on the <a href="https://github.com/yeatmanlab/roar-docs" target="_blank">GitHub repository</a>.</p>
```

Comment:
I think we should try to avoid html in the docs. Sometimes, it's unavoidable. For example, on the main landing page, I wanted some cards and didn't think I could do that with plain markdown. But in this case, I'd suggest going with just markdown. Something like:

```suggestion
This section provides technical documentation for researchers contributing to ROAR. To suggest edits, please open a pull request on the [GitHub repository][link_roar_docs_github].

[link_roar_docs_github]: https://github.com/yeatmanlab/roar-docs
```

Reply:
The only downside here is that you'll lose the centering. But I actually like this because then the formatting is consistent with the rest of the pages.